src/plugins/data/common/index_patterns/index_patterns/fixtures/logstash_fields.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { shouldReadFieldFromDocValues, castEsToKbnFieldTypeName } from '../../../../server';
+
+function stubbedLogstashFields() {
+  return [
+    //                                  |aggregatable
+    //                                  |      |searchable
+    // name               esType        |      |      |metadata       | subType
+    ['bytes', 'long', true, true, { count: 10 }],
+    ['ssl', 'boolean', true, true, { count: 20 }],
+    ['@timestamp', 'date', true, true, { count: 30 }],
```

Comment:
This fixture is being used in other plugins, any reason why we started duplicating them?

Reply:
Yes, `fixtures` folder doesn't have an owner. This file is used by `data` and `discover` plugins only. If you want to keep them in sync, we might need to consider exporting them from the `data` plugin.

Reply:
So what would be the rule of thumb of what goes into `src/fixtures`?

Reply:
> what goes into src/fixtures?

nothing. it should be removed eventually. global folders won't exist outside of a particular domain when we move to https://github.com/elastic/kibana/issues/71566